src/utils/darkMode.ts

```diff
@@ -0,0 +1,19 @@
+import { useCallback } from 'react';
+
+import { useColorMode } from '@chakra-ui/react';
+
+export const useDarkMode = () => {
+  const { colorMode, toggleColorMode, setColorMode } = useColorMode();
+
+  const colorModeValue = useCallback(
+    (lightValue, darkValue) => (colorMode === 'dark' ? darkValue : lightValue),
+    [colorMode]
+  );
+
+  return {
+    toggleColorMode,
+    setColorMode,
+    colorModeValue,
+    colorMode,
+  };
+};
```

Comment:
Maybe call this file `useDarkMode` and put it in a `hooks` folder ?

Reply:
Done :)